tests/res/fm/test_ecl_run_new.py

```diff
@@ -163,6 +163,23 @@ def test_failed_run_OK(self):
         ecl_config = Ecl100Config()
         run(ecl_config, ["SPE1_ERROR", "--version=2019.3", "--ignore-errors"])
 
+    @pytest.mark.equinor_test
+    @tmpdir()
+    def test_no_hdf5_output_with_ecl100(self):
+        self.init_eclrun_config()
+        shutil.copy(
+            os.path.join(self.SOURCE_ROOT, "test-data/local/eclipse/SPE1.DATA"),
+            "SPE1.DATA",
+        )
+        ecl_config = Ecl100Config()
+        # check that by default .h5 file IS NOT produced
+        run(ecl_config, ["SPE1.DATA", "--version=2019.3"])
+        self.assertFalse(os.path.exists("SPE1.h5"))
+
+        # check that with flag .h5 file IS produced
```

Comment:
Should we split this up into two tests so that the the two states are tested completely independently?

Reply:
sure, we can do that